tough/src/error.rs

```diff
@@ -478,6 +478,46 @@ pub enum Error {
 
     #[snafu(display("Role missing from snapshot meta: {}", name))]
     RoleNotInMeta { name: String },
+
+    #[snafu(display("Not able to Sign message for key-id {} : {}", key_id, source))]
+    KmsSign {
+        key_id: String,
+        source: rusoto_core::RusotoError<rusoto_kms::SignError>,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("Unable to create tokio runtime: {}", source))]
+    RuntimeCreation {
+        source: std::io::Error,
+        backtrace: Backtrace,
+    },
+
+    #[snafu(display("Signature is None"))]
+    SignatureNotFound,
+
+    #[snafu(display("Key Id: {} is not sign and verify key", key_id))]
+    NotSignVerifyKeyType { key_id: String },
+
+    #[snafu(display("Failed to run {}: {}", command_str, source))]
```

Comment:
Could we single quote the command string to make it easier to see it's a command?

`Failed to run '{}':...`